### ISSUE TITLE
Horizontal scrollbar is being displayed on the filters section of the menu

### ISSUE THREAD

**Issue by nerik** [Vizzuality/GlobalFishingWatch#806]
![screen shot 2017-10-20 at 09 42 19](https://user-images.githubusercontent.com/1583415/31810156-fb291868-b57a-11e7-895f-ec387b1d8afd.png)

@enriquetuya Could you please specify browser and OS versions ?

**Comment by enriquetuya** [comment]
@nerik Ubuntu 16 Version 62.0.3202.62 (Official Build) (64-bit)

**Comment by tiagojsag** [comment]
@enriquetuya would also be useful to know in which browser(s) you are experiencing this

**Comment by enriquetuya** [comment]
@tiagojsag check comment above.

**Comment by tiagojsag** [comment]
Hi @enriquetuya 

Just tested on FF and Chrome in both Windows 10 and Linux, and also on Edge, using the latest develop branch, and I cannot reproduce this issue. I believe Erik also tested on Mac, and also cannot find the issue.

**Comment by enriquetuya** [comment]
@tiagojsag @nerik this is only happening on Chrome on my Ubuntu machine. I've tried FF and works fine. 

Ubuntu: Ubuntu 16.04.3 LTS
Chrome: Version 62.0.3202.62 (Official Build) (64-bit)
More info: whatsmybrowser.org/b/YAHQ249

**Comment by tiagojsag** [comment]
@enriquetuya 
![image](https://user-images.githubusercontent.com/973964/32234295-ad14eda6-be5c-11e7-8d6b-cac697b77fa6.png)

I'll download a VM with Ubuntu and see if I can reproduce the issue, but I am not experiencing it on Linux (not Ubuntu) on a very similar chrome version

**Comment by enriquetuya** [comment]
@tiagojsag maybe something to do with the screen resolution?

**Comment by enriquetuya** [comment]
@tiagojsag @nerik a bit more of info:

![panel_width](https://user-images.githubusercontent.com/355597/32235033-21506332-be3d-11e7-804a-754d9b99607a.png)

**Comment by tiagojsag** [comment]
The closest I can get to that resolution without hacking Xorg is 1440x900 or 1920x1080. I just set up a VM with Ubuntu and Chrome, same exact versions you are using, and I cannot reproduce the issue. Can you try maybe on an external screen, with a more standard resolution?

![image](https://user-images.githubusercontent.com/973964/32236081-85fc16e0-be61-11e7-8bf2-2d142c09005b.png)

**Comment by tiagojsag** [comment]
Actually, my Ubuntu VM is using the .75 version of chrome, but i highly doubt that's the issue

**Comment by enriquetuya** [comment]
@nerik yes, this still happens to me. Maybe relates to: https://github.com/GlobalFishingWatch/map-client/issues/890?
